guide/03-the-gis/organization-inventory.ipynb

```diff
@@ -0,0 +1,2966 @@
+{
```

Comment:
<p>Being able to retrieve, display, analyze, and export the content within an organization Portal...  (capitalize product name)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>